server/statshouse/statshouse-manager.cpp

```diff
@@ -196,7 +196,8 @@ void StatsHouseManager::add_common_master_stats(const workers_stats_t &workers_s
     client.metric("kphp_by_host_version", true).tag(std::to_string(engine_tag_number)).write_count(1);
   }
 
-  client.metric("kphp_uptime").write_value(get_uptime());
+  static const auto dc_name = vk::singleton<ServerConfig>::get().get_dc_name();
+  client.metric("kphp_uptime").tag(dc_name).write_value(get_uptime());
```

Comment:
I think we want to have this tag for all our metric. Check [this](https://github.com/VKCOM/kphp/blob/bf6cd9a7d16e15c31a0b69b534c887305f935d7b/server/statshouse/statshouse-client.cpp#L11) out

Reply:
changed